pkg/hwinfo/hwinfo_other.go

```diff
@@ -194,3 +212,234 @@ func listDisplays() ([]*monitorInfo, error) {
 
 	return results, nil
 }
+
+// this part is extended version of github.com/shirou/gopsutil/cpu/cpu_linux.go
+// it need own implementation as gopsutil does return amount of threads not CPUs
```

Comment:
```suggestion
//  the custom implementation is required as gopsutil does return the number of threads, not CPUs
```

Reply:
done

Reply:
updated with `own implementation is required as gopsutil does return amount of threads instead of CPUs`